src/mtcp/mtcp_restart.c

```diff
@@ -184,11 +184,11 @@ MTCP_PRINTF("Attach for debugging.");
 {int x=1; while(x);}
 #endif
 
-// TODO(karya0): Remove vDSO checks after 2.4.0-rc3 release.
+// TODO(karya0): Remove vDSO checks after 2.4.0-rc3 release, and after testing.
 // Without mtcp_check_vdso, CentOS 7 fails on dmtcp3, dmtcp5, others.
 #define ENABLE_VDSO_CHECK
 // TODO(karya0): Remove this block and the corresponding file after sufficient
-// testing.
+// testing:  including testing for __i386__, __arm__ and __aarch64__
 #ifdef ENABLE_VDSO_CHECK
```

Comment:
@karya0: That's a good solution (keeping the code as is for now, and fixing it for 2.4.1).  Both @jiajuncao and @rohgarg have accounts on Stampede, and know how to observe this bug.  They can also give you a sub-account on Stampede, with instructions on observing the bug.  I assume you're busy enough that we can wait until after the 2.4.0 release for that.